AirCasting/Utils/NetworkChecker.swift

```diff
@@ -0,0 +1,49 @@
+// Created by Lunar on 08/07/2021.
+//
+
+import Foundation
+import Network
+
+protocol NetworkStatusPresenter {
+    var connectionAvailable: Bool { get }
+    func monitorNetwork()
```

Comment:
`monitorNetwork()` is not needed in this protocol anymore since no client should start monitoring by itself, it happens already in the init of implementation. It's only about the `connectionAvailable`